### PR TITLE
Added contact, about, footer

### PR DESCRIPTION
**Pull request by avinashkr1612**
Solved #11 #9 #6

### CODE REVIEW

**Comment by shivan1b** [comment]
You need to clean your git history. There should not be any "Merged bla bla with my branch" commits. Rebase and push only your changes properly.
And, please create a separate branch with a meaningful name. `master` should be your main branch, you should keep it even with the upstream.

File: _config.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ description: > # this means to ignore newlines until "baseurl:"
   LinuxChix is a community for women who like Linux and for anyone who wants to support women in computing. 
   We aim to promote FOSS primarily Linux and related projects among women and other minorities in tech. 
   We are a group of enthusiastic people ranging from students to professional experts willing to teach and foster the growth of Linux.
-baseurl: "/" # the subpath of your site, e.g. /blog
+baseurl: "" # the subpath of your site, e.g. /blog
```
Comment (utkarsh2102) [inline _config.yml:20]: This will not render the styling.
Please add this back.

File: _includes/footer.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<footer class="c-footer">
+<footer class="c-footer" style="background:#673AB7;color:#fff;" >
```
Comment (utkarsh2102) [inline _includes/footer.html:1]: Please add appropriate space b/w them so as to adhere to the styling guide.

**Review by utkarsh2102 — CHANGES_REQUESTED** [review]
Hey,
- Comments inline.
- Besides that, you need to create different branches for different issues.
- We do not accept PRs having more than 1 logical change or having more than 1 commit. Hence, requesting you to please squash commits and open 3 different PRs.
- Also, write a good commit message. Having a clean history as @shivan1b said, is really important.

Although we are thankful for the contribution, we cannot accept PRs like that.
Just make the requested changes and then you're good to go.

**Comment by utkarsh2102** [comment]
Hey,
We have changed the outlook of the website.
Please feel free to contact me if you still like to contribute.

**Pull request closed without merging**